lib/modules/manager/maven/extract.ts

```diff
@@ -464,6 +470,8 @@ function cleanResult(packageFiles: MavenInterimPackageFile[]): PackageFile[] {
     delete packageFile.parent;
     packageFile.deps.forEach((dep) => {
       delete dep.propSource;
+      //Add Registry From SuperPom
+      dep.registryUrls?.push(MAVEN_REPO);
```

Comment:
```suggestion
      dep.registryUrls!.push(MAVEN_REPO);
```
I think we know this is always set ? otherwise we would loose the registry.